CryptographicallyVerify.md

```diff
@@ -0,0 +1,76 @@
+<p align="left">
+  <img src="https://raw.githubusercontent.com/step-security/supply-chain-goat/main/images/Logo.png" alt="Step Security Logo" width="340">
+</p>
+
+# Tutorial: Cryptographically verify tools run as part of the CI/ CD pipeline
+
+## Summary of past incidents
+### SUNSPOT: An Implant in the Build Process
+In December 2020, the industry was rocked by the disclosure of a supply chain attack against SolarWinds, Inc. The supply chain attack was the result of inclusion of unauthorized malicious code during the build process. SUNSPOT is the name of the malware used to insert a backdoor into software builds of the SolarWinds Orion IT management product.
+
+One of the steps in this attack was carried out by SUNSPOT masquerading as a legitimate Windows Binary, and writing its logs in a fake VMWare log file. One of these files went by the name of `taskhostsvc.exe` which pretended to be a Windows Service Task Host Process.
```

Comment:
`One of the steps in this attack was carried out by SUNSPOT masquerading as a legitimate Windows Binary, and writing its logs in a fake VMWare log file. One of these files went by the name of `taskhostsvc.exe` which pretended to be a Windows Service Task Host Process` this part should be taken as a quote from the article as much as possible. e.g. you can state something like `SUNSPOT was identified on disk with a filename of taskhostsvc.exe (SHA256 Hash: c45c9bda8db1d470f1fd0dcc346dc449839eb5ce9a948c70369230af0b3ef168), and internally named taskhostw.exe by its developers. In this case SUNSPOT masquerades as a legitimate Windows Binary and replaces a source code file in the solution directory, with a malicious variant`